lib/actions.js

```diff
@@ -7,187 +7,295 @@
 
 var path = require('path');
 var g = require('strong-globalize')();
+var glob = require('glob');
 var fs = require('fs');
 var workspace = require('loopback-workspace');
 var Workspace = workspace.models.Workspace;
+var debug = require('debug')('loopback:generator:actions');
+var readChunk = require('read-chunk');
+var istextorbinary = require('istextorbinary');
+var memfs = require('mem-fs-editor');
 
-var actions = exports;
-
-// All actions defined in this file should be called with `this` pointing
-// to a generator instance
+module.exports = function ActionsMixin(baseClass) {
+  return class extends baseClass {
+    constructor(args, opts) {
+      super(args, opts);
+    }
+    // All actions defined in this file should be called with `this` pointing
+    // to a generator instance
 
-/**
+    /**
```

Comment:
extra space.

Reply:
I am not sure why the linting format is so weird here, but eslint complains about the old code...